### ISSUE TITLE
/debug/pprof and /debug/pprof/all endpoints are undocumented

### ISSUE THREAD

**Issue by lesam** [influxdata/docs-v2#3508]
The release notes for 2.0.5 say: 

> Add /debug/pprof/all HTTP endpoint to gather all profiles at once.

https://docs.influxdata.com/influxdb/v2.1/reference/release-notes/influxdb/#v205-general-availability-2021-04-27

But I don't see usage of this endpoint documented for 2.x

Usage:

```
> curl localhost:8086/debug/pprof/all --output pprof.tar.gz
> tar -xzvf pprof.tar.gz
> ls -l profiles
total 344
-rw-------  1 sarnold  staff  74559 31 Dec  1969 allocs.pb.gz
-rw-------  1 sarnold  staff    908 31 Dec  1969 block.pb.gz
-rw-------  1 sarnold  staff   4640 31 Dec  1969 goroutine.pb.gz
-rw-------  1 sarnold  staff  73867 31 Dec  1969 heap.pb.gz
-rw-------  1 sarnold  staff   2057 31 Dec  1969 mutex.pb.gz
-rw-------  1 sarnold  staff    280 31 Dec  1969 threadcreate.pb.gz
```

**Comment by lesam** [comment]
See also: https://go.dev/blog/pprof

**Comment by sanderson** [comment]
Related to #3413 and #3414. @jstirnaman Should we closes this and #3413 and use #3414 as an umbrella issue?

**Comment by lesam** [comment]
I'm fine with that - sorry I didn't see the existing issues, I just went looking for the docs in order to help an OSS user and realized they were missing.

**Comment by jstirnaman** [comment]
> Related to #3413 and #3414. @jstirnaman Should we closes this and #3413 and use #3414 as an umbrella issue?
Duplicate of #3414
